src/Microsoft.AspNetCore.Razor.VSCode/src/CSharp/CSharpProjectedDocumentContentProvider.ts

```diff
@@ -3,28 +3,33 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 
-import * as vscode from 'vscode';
+import { IEventEmitterFactory } from '../IEventEmitterFactory';
 import { IRazorDocumentChangeEvent } from '../IRazorDocumentChangeEvent';
+import { IRazorDocumentManager } from '../IRazorDocumentManager';
 import { RazorDocumentChangeKind } from '../RazorDocumentChangeKind';
-import { RazorDocumentManager } from '../RazorDocumentManager';
 import { getUriPath } from '../UriPaths';
+import * as vscode from '../vscodeAdapter';
```

Comment:
The extra changes to this document are to enable it to be tested. Essentially moved it and the Html variant over to use the vscodeAdapter so we can unit test it properly.